manifests/server/passwd.pp

```diff
@@ -5,6 +5,15 @@
   $group             = $postgresql::server::group
   $psql_path         = $postgresql::server::psql_path
   $port              = $postgresql::server::port
+  $database          = $postgresql::server::default_database
+
+  # psql will default to connecting as $user if you don't specify name
+  # dbname option could probably be used unconditionally assuming it is in all supported versions of psql
+  $datbase_user_same = $database == $user
+  $dboption = $datbase_user_same ? {
```

Comment:
I had ($database == $user) in the selector before but I think it made the travis tests fail on Ruby 1.9. I didn't see any other selectors in the module that used expressions. The expression was working for me and in the other non 1.9 tests.